bin/main.rb

```diff
@@ -2,8 +2,10 @@
 require_relative '../lib/board.rb'
 require_relative '../lib/player.rb'
 
-board=Board.new
+board = Board.new
 
-players=[Player.new,Player.new]
+players = [Player.new, Player.new]
```

Comment:
It would be better if you declare separately player_1 and player_2 rather than putting them in an array and passing this array as an argument to Game. 

What if later on you would like to make a 1-player mode here where a human player can play with the computer? Doing it like this will make it hard for you to extend the game's features later on.